ajo/personalization-server-side/README.md

```diff
@@ -0,0 +1,210 @@
+3. When the page is requested from the app server, an event is sent to the [interactive data collection endpoint](https://experienceleague.adobe.com/docs/experience-platform/edge-network-server-api/data-collection/interactive-data-collection.html?lang=en) to fetch personalization content. This sample app makes use of some helper methods to simplify building and sending requests to the API (see [aepEdgeClient.js](../../common/aepEdgeClient.js)). But the request is simply a `POST` with a payload that contains an event and query. The cookies (if available) from the prior step are included with the request in the `meta>state>entries` array.
+
+```javascript
+fetch(
+  "https://edge.adobedc.net/ee/v2/interact?dataStreamId=abc&requestId=123",
+  {
+    headers: {
+      accept: "*/*",
+      "accept-language": "en-US,en;q=0.9",
+      "cache-control": "no-cache",
+      "content-type": "text/plain; charset=UTF-8",
+      pragma: "no-cache",
+      "sec-fetch-dest": "empty",
+      "sec-fetch-mode": "cors",
+      "sec-fetch-site": "cross-site",
+      "sec-gpc": "1",
+      "Referrer-Policy": "strict-origin-when-cross-origin",
+      Referer: "https://localhost/",
+    },
+    body: JSON.stringify({
+      event: {
+        xdm: {
+          eventType: "decisioning.propositionFetch",
+          web: {
+            webPageDetails: {
+              URL: "https://localhost/",
+            },
+            webReferrer: {
+              URL: "",
+            },
+          },
+          identityMap: {
+            FPID: [
+              {
+                id: "xyz",
+                authenticatedState: "ambiguous",
+                primary: true,
+              },
+            ],
+          },
+          timestamp: "2022-06-23T22:21:00.878Z",
+        },
+        data: {},
+      },
+      query: {
+        identity: {
+          fetch: ["ECID"],
+        },
+        personalization: {
+          schemas: [
+            "https://ns.adobe.com/personalization/default-content-item",
+            "https://ns.adobe.com/personalization/html-content-item",
+            "https://ns.adobe.com/personalization/json-content-item",
+            "https://ns.adobe.com/personalization/redirect-item",
+            "https://ns.adobe.com/personalization/dom-action",
+          ],
+          surfaces: ["web://127.0.0.1/#sample-json-offer"]
+        },
+      },
+      meta: {
+        state: {
+          domain: "localhost",
+          cookiesEnabled: true,
+          entries: [
+            {
+              key: "kndctr_XXX_AdobeOrg_identity",
+              value: "abc123",
+            },
+            {
+              key: "kndctr_XXX_AdobeOrg_cluster",
+              value: "or2",
+            },
+          ],
+        },
+      },
+    }),
+    method: "POST",
+  }
+).then((res) => res.json());
+```
+
+4. The target offer from the form-based activity is read from the response and used when producing the HTML response.
```

Comment:
@krishnanbalasubramanyam  I think that your example is based on a JSON offer, right? then the message should be something like: "The JSON offer from the code-based experience action is read from the response and used when producing the JSON response."

Reply:
Right. Updated it.
The JSON response is producing the HTML response eventually on the page